pkgs/tools/misc/shim/default.nix

```
@@ -24,11 +25,20 @@ in stdenv.mkDerivation rec {
 
   buildInputs = [ elfutils ];
 
-  env.NIX_CFLAGS_COMPILE = toString [ "-I${toString elfutils.dev}/include" ];
+  env.NIX_CFLAGS_COMPILE = toString ([
+    "-I${toString elfutils.dev}/include"
```

```suggestion
    "-I${elfutils.dev}/include"
```
Isn't this just working like this?